### DISCUSSION TITLE
QInput error overlapps next field

### DISCUSSION THREAD

**Discussion by thomasemmerich** [quasarframework/quasar#15063] [General - Components / Directives / etc] [answered]
Hi,  I have the following issue: If I have a bigger error message, the error message overlapps the next input field.
`        <q-input
          square
          filled
          v-model="form.zipcode"
          bottom-slots
          type="number"
          :error-message="validators.zipcode"
          :error="!!validators.zipcode"
        />`
`        <q-select
          square
          filled
          emit-value
          bottom-slots
          map-options
          v-model="form.typ"
          :options="types"
          :error-message="validators.typ"
          :error="!!validators.typ"`

![Screenshot 2022-12-06 160119](https://user-images.githubusercontent.com/15215440/205947266-7bf17fb9-3528-4534-852a-b1f85fb6aedc.png)

**Comment by metalsadman** [comment]
well that's a very long error message, inspect the element and prolly add margin to it.

**Comment by thomasemmerich** [comment, marked as answer]
My Solution is now a seperate `<p>` under the input like this:
```
<q-input
    square
    filled
    clearable
    :error="validators.theerror"
  />
<p v-if="validators.theerror" class="p-error">
  {{ validators.theerror}}
</p>
```
